e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-house-search.spec.js

```diff
@@ -226,4 +228,19 @@ context('Insurance - Your business - Company details page - company house search
       cy.checkText(partials.yourBusinessSummaryList[COMPANY_SIC].value(), DEFAULT.EMPTY);
     });
   });
+
+  describe('when going back to company details page after searching for company house and pressing continue', () => {
+    it('should take ytou back to company details page', () => {
+      cy.navigateToUrl(url);
+
+      cy.keyboardInput(companyDetails.companiesHouseSearch(), COMPANIES_HOUSE_NUMBER);
+      companyDetails.companiesHouseSearchButton().click();
```

Comment:
Just searched the codebase for line 236, 18 instances of this. Could we create or use a cypress command for this? `cy.completeAndSubmitCompaniesHouseSearchForm()`?